backend/geonature/utils/config_schema.py

```diff
@@ -255,9 +255,78 @@ class GnFrontEndConf(Schema):
 
 
 class Synthese(Schema):
+    # --------------------------------------------------------------------
+    # SYNTHESE - SEARCH FORM
     AREA_FILTERS = fields.List(
         fields.Dict, load_default=[{"label": "Communes", "type_code": "COM"}]
     )
+    # Nombre de résultat à afficher pour la rechercher autocompleté de taxon
+    TAXON_RESULT_NUMBER = fields.Integer(load_default=20)
+    # Afficher ou non l'arbre taxonomique
+    DISPLAY_TAXON_TREE = fields.Boolean(load_default=True)
+    # Ajouter le filtre sur l'observers_txt en ILIKE sur les portée 1 et 2 du CRUVED
+    CRUVED_SEARCH_WITH_OBSERVER_AS_TXT = fields.Boolean(load_default=False)
+    # Switch the observer form input in free text input (true) or in select input (false)
+    SEARCH_OBSERVER_WITH_LIST = fields.Boolean(load_default=False)
```

Comment:
Comme ceci peut-être ?
```suggestion
    SEARCH_OBSERVER_FREE_TEXT = fields.Boolean(load_default=False)
```

Reply:
Idem, le paramètre `SEARCH_OBSERVER_WITH_LIST` est un paramètre existant qui a juste été déplacé.